File: exercicios/1930/1930.py
n = input().split()

t1 = int(n[0])
t2 = int(n[1])
t3 = int(n[2])
t4 = int(n[3])
T = 0

if t1 > 0:
    t1 = t1 - 1

if t2 > 0:
    t2 = t2 - 1
    t1_2 = t2 + t1

if t3 > 0:
    t3 = t3 - 1
    t1_2_3 = t1_2 + t3

if t4 > 0:
    T = t4 + t1_2_3

print(T)

File: exercicios/JavaScript/1930/1930.js
var input = require('fs').readFileSync('stdin','utf8');
var lines = input.split('\n');

var val = lines.shift().split(' ')

var t1 = parseInt(val[0])
var t2 = parseInt(val[1])
var t3 = parseInt(val[2])
var t4 = parseInt(val[3])

var total = 0

total = (t1 - 1) + (t2 - 1) + (t3 - 1) + t4
console.log(total)